src/controllers/controllerpresetfilehandler.cpp

```diff
@@ -78,6 +78,8 @@ void ControllerPresetFileHandler::parsePresetInfo(const QDomElement& root,
         return;
     }
 
+    int iControllerEngineVersion = root.attribute("controllerEngineVersion", 0).toInt();
```

Comment:
Shouldn't the default be 1, not 0?

Reply:
Actually it's the same, both numbers will work with the ControllerEngine version 1. But setting 0 let us differentiate between presets that specified version 1 and presets that did not specify anything. Might be useful in a future? Since it does not make a difference I think we can leave the 0.

Reply:
:man_shrugging: